src/Parser/Monk/Windwalker/Modules/Spells/TouchOfDeath.js

```diff
@@ -45,14 +46,29 @@ class TouchOfDeath extends Analyzer {
     this.expectedGaleBurst = 0;
     const masteryPercentage = this.statTracker.currentMasteryPercentage;
     const versatilityPercentage = this.statTracker.currentVersatilityPercentage;
-    this.expectedBaseDamage = (event.maxHitPoints * TOUCH_OF_DEATH_HP_SCALING + meridianStrikesDamage(this.selectedCombatant.traitsBySpellId[SPELLS.MERIDIAN_STRIKES.id]).damage) * (1 + masteryPercentage) * (1 + versatilityPercentage);
+
+    // Calculate Meridian Strikes Modifier Damage if combatant has MS traits
+    const merStrikesTraits = 
+      this.selectedCombatant.traitsBySpellId[SPELLS.MERIDIAN_STRIKES.id];
+
+    const merStrikesMod = 
+      (merStrikesTraits === undefined) ? 0.0 : meridianStrikesDamage(
+      this.selectedCombatant.traitsBySpellId[SPELLS.MERIDIAN_STRIKES.id]).damage;   
+
+    this.expectedBaseDamage =
```

Comment:
This code style makes the code very long

Reply:
You prefer the 281 long single line?

Reply:
Absolutely, most devs have a wide screen monitor, making the vertical space is limited

Reply:
I *strongly* disagree.. long lines are terrible for human code readability, in fact, almost every programming language style-guide enforce a 80 character column limit. To my knowledge, so do all major JavaScript ES6 style guides.

Reply:
That's a poor argument imo. Relying on client-side tools instead of writing readable code, by that logic you may as well not use any style guides at all. A better alternative is to introduce linting to the project to try and enforce consistentency by linting against battle tested linter guides (e.g. Google ES6, React ES6, AirBnb etc.).

Reply:
Unfortunately those read to code that is harder to read, and the max column limit (especially 80) is based on very outdated ideas that do not fit the project

Reply:
Sorry, very outdated? Literally every major 2018 JS Style Guide spec has the 80 column limit (100 absolute max) and if you look around at code samples you are not likely to find many 100+ lines. If you don't want that in this project for some reason that's one thing, but to say that it is "outdated" is just incorrect.

Reply:
iirc the 80 column limit dates back to time cards and have hardly changed even though we've begun using widescreen monitors up to 4 thousand pixels in width. With soft wrapping and other client side tools for your IDE, you can set it up locally the way you want. You can have things use the optimal space available and be as easy to read as the screen size you allocate, even if you have just one file open, multiple in a split screen or multiple windows dragged around the desktop. Those tools don't work well if the code already has every single variable on a separate line (how this becomes more readable is beyond me). 

But still anyone who takes ownership of a spec can decide for the most part how to do it themselves. We'd prefer consistency across the codebase, but in the end you are free to choose how to do opinionated/preference things that aren't enforced by eslint rules. It also complicates things when other people would want to help out, or when working on shared components, so long story short consistency is encouraged.

Reply:
You're right, it dates back to IBM Punchcards which had 80 columns. As for why it is still the industry standard in 2018, many discussions like ours now have taken place and the general consensus seems to be that shorter lines are still a lot more readable for humans, hence why the industry still enforces line length rules of less than 80-100 characters.

The tools you mentioned (soft-wrap in IDEs etc) are not usually encouraged because they typically simply cut lines on the last keyword that would violate the line-length, so they often make the readability worse. 

At the end of the day, it is a matter of personal opinion as you say, and if you want to enforce long lines in this project that's your prerogative, what bothered me was that you flagged it in the PR and then said that this was outdated practice, when in fact it is the industry standard for almost every programming language, including JavaScript ES6. As a professional in the industry, I would be crucified if I submitted a 281 char code line.

Either way, let's move on ;)